docs/source/sections/Installation.rst

```diff
@@ -10,53 +10,42 @@ Windows
 #######
 The PolarRoute software requires GDAL files to be installed. The PolarRoute software can be installed on Windows by running one of the two following commands.
 
-Github:
-::
-    
-    pip install pipwin
-    pipwin install gdal
-    pipwin install fiona
-    python setup.py install
+.. note:: 
+    We assume a version of Windows 10 or higher, with a working version of Python 3.9 including pip installed. 
+    We recommend installing PolarRoute into a virtual environment.
 
-Pip: 
+Requirements:
 ::
 
-    pip install pipwin
+    pip install pipwin # pipwin is a package that allows for easy installation of windows binaries
     pipwin install gdal
     pipwin install fiona
-    pip install polar-route
-
-**NOTE**
 
-For GDAL (one of the required dependencies for the project), you may try the following commands to avoid any installation issues:
-::
-   
-    pipwin install GDAL
     pip install -r requirements.txt
-    python setup.py install
-
-
-Linux/MacOS
-###########
-
-The PolarRoute software can be installed on Linux/MacOS by running one of the two following commands.
 
 Github:
 ::
 
+    git clone https://https://github.com/antarctica/PolarRoute.git
     python setup.py install
 
 Pip: 
 ::
 
     pip install polar-route
 
+Linux/MacOS
+###########
 
-**NOTE**
+The PolarRoute software can be installed on Linux/MacOS by running one of the two following commands.
 
-For GDAL (one of the required dependencies for the project), you may try the following commands to avoid any installation issues:
-::
+Requirements:
+
+.. note::
+    For GDAL (one of the required dependencies for the project), you may try the following 
+    commands to avoid any installation issues:
 
+::
    
     sudo add-apt-repository ppa:ubuntugis/ppa && sudo apt-get update
     sudo apt-get update
```

Comment:
It's a little harder for me to check if it's really needed since I'm running Windows not Linux as my OS. Would someone whose running Linux (@ayat-khairy, @hjabbot, @gecoombs) be able to double check?

Reply:
should still work if we got rid of `sudo apt-get update` line.

Reply:
Yeah, Harry has updated this now in 83b0869 :slightly_smiling_face: